client/my-sites/activity/activity-log-item/index.jsx

```diff
@@ -151,6 +154,34 @@ class ActivityLogItem extends Component {
 		} );
 	}
 
+	getPLainTextActivityDescription() {
```

Comment:
@enejb Thanks for this! :)

Reply:
Camel case looks off here: `PLain`->`Plain`

Reply:
Thanks for this catch @sirreal 👍